libraries/helpers.rb

```diff
@@ -34,7 +34,12 @@ module ChefVaultCookbook
   # @param [String] id Identifier of the data bag item to load.
   def chef_vault_item(bag, id)
     if ChefVault::Item.vault?(bag, id)
-      ChefVault::Item.load(bag, id)
+      auth_params = {}
+      if node['chef-vault']['node_name']
```

Comment:
Its personal preference for me, but I would pass in the hash as an optional parameter to the method.